docs/sources/setup/install/helm/deployment-guides/azure.md

```diff
@@ -0,0 +1,588 @@
+---
+title: Deploy the Loki Helm chart on Azure
+menuTitle: Deploy on Azure
+description: Installing the Loki Helm chart on Azure.
+keywords: 
+---
+
+# Deploy the Loki Helm chart on Azure
+
+This guide shows how to deploy a minimally viable Loki in **microservice** mode on Azure using the Helm chart. To run through this guide, we expect you to have the necessary tools and permissions to deploy resources on Azure, such as:
+
+- Full access to AKS (Azure Kubernetes Service)
+- Full access to Azure Blob Storage
+- Sufficient permissions to create federated credentials and roles in Azure AD (Active Directory)
+
+There are three primary methods for deploying Loki on Azure:
+1. Hard coding a connection string - this is the simplest method but is not recommended for production environments.
+2. Manged Identity
+3. Federated Token
```

Comment:
Since these aren't steps, change to bullets.
```suggestion
There are three primary methods for deploying Loki on Azure:

- Hard coding a connection string - this is the simplest method but is not recommended for production environments.
- Manged Identity
- Federated Token
```